src/mlpack/bindings/tests/clean_memory.cpp

```diff
@@ -2,7 +2,7 @@
  * @file bindings/tests/clean_memory.cpp
  * @author Ryan Curtin
  *
- * Delete any pointers held by the CLI object.
+ * Delete any pointers held by the IO object.
```

Comment:
I am not sure, whether these CLI's are ignored or these are intentional. And If they are ignored should I open a different PR or we are good here with these changes?

Reply:
Yes, it looks to me like there were some things that were accidentally missed.  Do you want to open a PR to the main repository with those things?  Alternately I can do it, just let me know---I can see that these were missed, and also the names of the utility functions in `src/mlpack/bindings/julia/mlpack/cli.jl.in`.  I'm not sure if the Python utility functions need to be adapted too; I didn't check.  Anyway let me know what you'd like to do.

Reply:
I will open a PR soon for the same.

Reply:
Thanks!